config/resources/lpdc.json

```diff
@@ -0,0 +1,644 @@
+        }
+      },
+      "relationships": {
+        "hasLegalResource": {
+          "target": "conceptual-public-services",
+          "predicate": "m8g:hasLegalResource",
+          "cardinality": "many",
+          "inverse": true
+        }
+      },
+      "features": [
+        "include-uri"
+      ],
+      "new-resource-base": "http://data.lblod.info/id/legal-resource/"
+    },
+    "contact-points": {
+      "class": "schema:ContactPoint",
+      "attributes": {
+        "email": {
+          "type": "string",
+          "predicate": "schema:email"
+        },
+        "telephone": {
+          "type": "string",
+          "predicate": "schema:telephone"
+        },
+        "fax": {
+          "type": "string",
+          "predicate": "schema:faxNumber"
+        },
+        "website": {
+          "type": "uri",
+          "predicate": "foaf:page"
+        },
+        "type": {
+          "type": "string",
+          "predicate": "schema:contactType"
+        },
+        "opening-hours": {
+          "type": "string",
+          "predicate": "schema:openingHours"
+        }
+      },
+      "relationships": {
+        "hasContactPoint": {
+          "target": "conceptual-public-services",
+          "predicate": "m8g:hasContactPoint",
+          "cardinality": "many",
+          "inverse": true
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/id/contact-punten/"
+    },
+    "documents": {
+      "class": "dcat:Distribution",
+      "attributes": {
+        "description": {
+          "type": "string",
+          "predicate": "dct:description"
+        },
+        "identifier": {
+          "type": "string",
+          "predicate": "dct:identifier"
+        },
+        "name": {
+          "type": "string",
+          "predicate": "dct:title"
+        }
+      },
+      "relationships": {
+        "hasAttachment": {
+          "target": "conceptual-public-services",
+          "predicate": "lpdcExt:attachment",
+          "cardinality": "many",
+          "inverse": true
+        }
+      },
+      "features": [
+        "include-uri"
+      ],
+      "new-resource-base": "http://data.lblod.info/id/requirement/"
+    },
+    "locations": {
+      "class": "dct:Location",
+      "relationships": {
+        "hasAttachment": {
+          "target": "conceptual-public-services",
+          "predicate": "dct:spatial",
+          "cardinality": "many",
+          "inverse": true
+        }
+      },
+      "features": [
+        "include-uri"
+      ],
+      "new-resource-base": "http://data.lblod.info/id/location/"
+    },
+    "lifecycle-statuses": {
+      "class": "code:LifecycleStatus",
+      "attributes": {
+        "label": {
+          "type": "string",
+          "predicate": "skos:prefLabel"
+        }
+      },
+      "new-resource-base": "http://lblod.data.gift/concepts/"
+    },
+    "sectors": {
+      "TODO" : "NACE codes? cfr dienstencataloog AP",
+      "class": "code:Sector",
```

Comment:
Could it be that there is no data for this yet?

Reply:
Is there any data for it in the Sharepoint files, or do we need to request that somewhere? It's used in some table columns and filters so it has an impact on the frontend.

Reply:
No data in the sharepoint files

Reply:
Let me know if something should be mocked